quadriga/src/main/java/edu/asu/diging/quadriga/api/v1/GetCollectionTriplesApiController.java

```diff
@@ -0,0 +1,78 @@
+package edu.asu.diging.quadriga.api.v1;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.asu.diging.quadriga.config.web.TokenInfo;
+import edu.asu.diging.quadriga.core.aspect.annotation.InjectToken;
+import edu.asu.diging.quadriga.core.exceptions.CollectionNotFoundException;
+import edu.asu.diging.quadriga.core.exceptions.InvalidObjectIdException;
+import edu.asu.diging.quadriga.core.model.Collection;
+import edu.asu.diging.quadriga.core.model.MappedTripleGroup;
+import edu.asu.diging.quadriga.core.model.MappedTripleType;
+import edu.asu.diging.quadriga.core.service.CollectionManager;
+import edu.asu.diging.quadriga.core.service.MappedTripleGroupService;
+import edu.asu.diging.quadriga.core.service.MappedTripleService;
+
+@Controller
+public class GetCollectionTriplesApiController {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    @Autowired
+    private CollectionManager collectionManager;
+
+    @Autowired
+    private MappedTripleService mappedTripleService;
+
+    @Autowired
+    private MappedTripleGroupService mappedTripleGroupService;
+
+    @InjectToken
+    @GetMapping(value = "/api/v1/collection/{collectionId}/triples", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<String> getTriples(@PathVariable("collectionId") String collectionId, TokenInfo tokenInfo) {
+        try {
+
+            Collection collection = collectionManager.findCollection(collectionId);
+
+            if (collection.getApps() == null || collection.getApps().isEmpty()
+                    || !collection.getApps().contains(tokenInfo.getClient_id())) {
+                return new ResponseEntity<>(HttpStatus.UNAUTHORIZED);
+            }
```

Comment:
that shouldn't stay the only method that access collection and needs a permission check, so it should be in an aspect.